spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -16,8 +16,9 @@
 	<packaging>pom</packaging>
 
 	<properties>
-		<google-cloud-bom.version>0.50.0-alpha</google-cloud-bom.version>
-		<cloud-trace-java.version>0.5.0</cloud-trace-java.version>
+		<google-cloud-bom.version>0.51.0-alpha</google-cloud-bom.version>
+		<google-auth.version>0.9.1</google-auth.version>
```

Comment:
nit, but should we call this `google-auth-library-oauth2-http.version`, not to confuse with other Google auth libraries?